### DISCUSSION TITLE
Latest stable version

### DISCUSSION THREAD

**Discussion by wheremyfoodat** [wheremyfoodat/Panda3DS#226] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/wheremyfoodat/Panda3DS/releases/tag/v0.4-beta'>Latest stable version</a>.</em>

**Comment by wheremyfoodat** [comment]
@OFFTKP 
![image](https://github.com/wheremyfoodat/Panda3DS/assets/44909372/7c799bfa-a268-4756-a610-d07181e86e21)

**Comment by georgemoralis** [comment]
let me test :D

**Reply by wheremyfoodat** [reply]
No Resident Evil
![image](https://github.com/wheremyfoodat/Panda3DS/assets/44909372/e13905bd-15b5-49d4-bf2e-d9a5e01573ba)

**Comment by OFFTKP** [comment]
Peach when he hasn't released a new stable version in 27 hours:
![image](https://github.com/wheremyfoodat/Panda3DS/assets/21157395/ee304c30-ebb7-4293-84ae-3caf897bb5c9)

**Reply by wheremyfoodat** [reply]
I am unstable 
![image](https://github.com/wheremyfoodat/Panda3DS/assets/44909372/a03a9ba6-a688-4e56-98c2-061aaead8068)

**Reply by OFFTKP** [reply]
When github allows you to reply while the image is still uploading:

![Uploading image.png…]()

**Reply by wheremyfoodat** [reply]
![Uploading image.png…]()

**Comment by georgemoralis** [comment]
if you want to complain come to discord

**Comment by georgemoralis** [comment]


**Reply by wheremyfoodat** [reply]
Brb advertising under your discussion section

**Reply by georgemoralis** [reply]
i was talking to paris , παρασυρθηκα...

**Reply by OFFTKP** [reply]
![image](https://github.com/wheremyfoodat/Panda3DS/assets/21157395/5efc75ee-a1bd-4f02-904b-04001749cd68)

**Reply by wheremyfoodat** [reply]
ΑΧΑΧΑΨΑΨΑΧΑΧΑΧΧΑ
